plugins/woocommerce-blocks/tests/e2e/tests/checkout/additional-fields.shopper.block_theme.spec.ts

```diff
@@ -805,11 +805,11 @@ test.describe( 'Shopper → Additional Checkout Fields', () => {
 
 			// Check text inputs in edit mode match the expected values.
 			const shippingGovIdInput = checkoutPageObject.page.getByLabel(
-				'Government ID *',
+				'Government ID',
 				{ exact: true }
 			);
 			const shippingConfirmGovIdInput =
-				checkoutPageObject.page.getByLabel( 'Confirm government ID *', {
+				checkoutPageObject.page.getByLabel( 'Confirm government ID', {
```

Comment:
```suggestion
				'Confirm government ID *',
```

Due to `{ exact: true }`, removing the asterisk causes this e2e test to fail.